cwms-radar-client/src/test/resources/radar/v1/json/outlets.json

```diff
@@ -0,0 +1,54 @@
+[
+  {
+    "project-id": {
+      "office-id": "SPK",
+      "name": "BIGH"
+    },
+    "location": {
+      "office-id": "SPK",
+      "name": "BIGH-TG1",
+      "latitude": 0.0,
+      "longitude": 0.0,
+      "active": true,
+      "public-name": "BIGH-TG1",
+      "timezone-name": "UTC",
+      "location-kind": "Outlet",
+      "horizontal-datum": "NAD84",
+      "vertical-datum": "NAVD88"
+    },
+    "rating-category-id": {
+      "office-id": "SPK",
+      "name": "Rating"
+    },
+    "rating-group-id": {
+      "office-id": "SPK",
+      "name": "Rating-BIGH-TG1"
+    }
+  },
+  {
+    "project-id": {
+      "office-id": "SPK",
+      "name": "BIGH-TEST12"
```

Comment:
This is the project location not the outlet location.  Please switch it to BIGH and update the outlet location at line 35.  Also, don't use TEST12, go with TG2, or something else more appropriate for a gate

Reply:
Renamed values to match standard